sdk/communication/azure-communication-identity/azure/communication/identity/aio/_communication_identity_client_async.py

```diff
@@ -87,8 +85,9 @@ async def create_user(self, **kwargs) -> "CommunicationUserIdentifier":
         return CommunicationUserIdentifier(identity_access_token.identity.id, raw_id=identity_access_token.identity.id)
 
     @distributed_trace_async
-    async def create_user_and_token(
-        self, scopes: List[Union[str, "CommunicationTokenScope"]], **kwargs
+    async def create_user_and_token( # pylint: disable=C4758
+        self, scopes: List[Union[str, CommunicationTokenScope]],
```

Comment:
```suggestion
        self, scopes: List[Union[str, CommunicationTokenScope]],*, token_expires_in: timedelta
```

Is there a reason to not do this?

Reply:
@l0lawrence these are public APIs and it might bring breaking changes for the customers. Even when I switched to the keyword-readonly values our tests started to fail, so that's why I kept the param in kwargs. Do you think it won't introduce any breaking changes?

Reply:
It should not introduce any breaking changes, when you add it to the method signature I believe it would be an keyword-only arg with a default value of None, and then in the code below on line 104 you would need to remove the kwargs.pop()

Reply:
Thanks, fixed!